app/.gitignore

```diff
@@ -1,2 +1,3 @@
 node_modules
-dist
\ No newline at end of file
+dist
+stats.html
```

Comment:
direct path? Seems a bit generic.

Reply:
this is the gitignore nested under app so it has an implied nested path - it's what gets generated by vite dev i can look into nesting it via the visualizer plugin config